### ISSUE TITLE
-S (sourcename not working)

### ISSUE THREAD

**Issue by VCR-2** [pulsejets/check_wmi_eventid#2]
When using -S <source name>, it doesnt works

**Comment by rojobull** [comment]
Yes. The problem is in the line that reads in the arguments. The problem is that a colon is missing in between the Sw. This basically ignores the source name you pass it. Simply add a colon and it will work. I am putting in a pull request with some other changes and will add this to it.

Current:
while getopts "hH:u:p:f:l:t:e:s:Sw:c:m:W:C:O:U:dv" OPTION
Changed to:
while getopts "hH:u:p:f:l:t:e:s:S:w:c:m:W:C:O:U:dv" OPTION

**Comment by VCR-2** [comment]
Thanks a lot! It works!
